Page Object/Book/loginAPI.ts

```diff
@@ -0,0 +1,95 @@
+import { baseAPI } from "../baseAPI";
+export class LoginApi extends baseAPI {
+    private readonly baseUrl: string;
+    constructor (page) {
+        super (page)
+        this.baseUrl = 'https://demoqa.com';
+    }
+
+    async registerUserRequest(userName: string, password: string) {
+        const response = await this.page.request.post(`${this.baseUrl}/Account/v1/User`, {
+            data: {"userName":userName,"password":password}
+            });
+            const body = await this.GetResponceBody(response)
+            console.log(body.userID)
+            return body.userID;
+    }
+
+    async generateToken(userName: string, password: string) {
+        const response = await this.page.request.post(`${this.baseUrl}/Account/v1/GenerateToken`, {
+            data: {"userName":userName,"password":password}});
```

Comment:
if you have the same key and value name, then you can just use one word. In this situation `data: {userName, password}`